modules/cbelasticsearch/models/IndexBuilder.cfc

```diff
@@ -138,6 +145,24 @@ component
 		return this;
 	}
 
+	/**
+	 * Update the mapping on an existing index.
+	 * 
+	 * @name 		{String}	Index name. Defaults to the default index set in configuration.
+	 * @properties 	{Struct}	Index mapping. Defines the fields and types used in the index.
+	 * @settings 	{Struct}	Key/value struct of index settings such as `number_of_shards`.
+	 */
+	IndexBuilder function update( string name, any properties, struct settings ) {
+		if ( !isNull( arguments.settings ) && !structIsEmpty( arguments.settings ) ){
```

Comment:
@michaelborn : I need a reply on this one as to why the change here

Reply:
@jclausen I don't remember off the top of my head why I made this change, but I do think i was getting _some_ sort of error when attempting to update index settings. I was not aware that some settings can be changed on an existing index - perhaps I was trying to update a locked setting, ran into an Elasticsearch error, and promptly decided to add a throw to avoid confusion?
I'll try pulling this down and re-running my code, if I can find where I was working with it. I may have been trying to update the refresh interval on an index during some data imports.